packages/@okta/vuepress-site/docs/reference/architecture-center/directory-coexistence/lab-tutorial-2-ldap-server/index.md

```diff
@@ -0,0 +1,575 @@
+
+1. Assign the admin role to your new user:
+
+   1. Select **Admins > Add administrator**.
+   2. Locate the **Administrator assignment by admin** screen and set **Admin** to **LDAP User (ldap-user@example.com)**.
+   3. Select **groups-user-reader** for the **Role** field.
+   4. Select **Users and Groups** for the **Resource set** field.
+   5. Click **Save Changes**.
+
+      <div class="full border">
+
+      ![assign admin role settings](/img/architecture/directory-coexistence/ldap-to-okta-assign-admin-role-to-ldap-user.png)
+
+      </div>
+
+   6. Refresh the **Administrator** page to see your new administrator, **LDAP User**.
+
+You can test the user's access using the following `ldapsearch` command in the console. Remember to replace `${OKTA_DOMAIN_NAME}` with your Okta Org name and `${OKTA_LDAP_PASSWORD}` with the password that you set for the `OKTA_LDAP_PASSWORD` variable in the `okta-reference-coexistence-ldap-example/.env` file first.
+
+```bash
+docker run bitnami/openldap:2 ldapsearch -x -b "ou=groups,dc=dev-133337,dc=okta,dc=com" -H ldaps://${OKTA_DOMAIN_NAME}.ldap.okta.com:636 -D uid=ldap-user,dc=dev-133337,dc=okta,dc=com -w '${OKTA_LDAP_PASSWORD}'
+```
+
+The result will be a list of groups with uniqueMember attributes, including your admin user and the three users you imported from the LDAP server into Okta. A portion of the listing will be similar to the following:
+
+```txt
+uniqueMember: uid=ldap-user@example.com,ou=users,dc=dev-133337,dc=okta,dc=com
+uniqueMember: uid=user01@example.com,ou=users,dc=dev-133337,dc=okta,dc=com
+uniqueMember: uid=user02@example.com,ou=users,dc=dev-133337,dc=okta,dc=com
+uniqueMember: uid=user03@example.com,ou=users,dc=dev-133337,dc=okta,dc=com
+```
+
+### Rebuild the application to incorporate your changes
+
+1. Rebuild the application:
+
+   ```bash
+   docker compose build
+   ```
+
+2. Start the application:
+
+   ```bash
+   docker compose up
+   ```
+
+3. Open a private/incognito window and navigate to the same URL as before: `http://localhost:8080`.
+
+4. Sign in with the imported user's email address and password: `user01@example.com` and `password1`.
+
+   <div class="half border">
+
+   ![please sign in dialog with filled in username and password](/img/architecture/directory-coexistence/ldap-to-okta-signin-dialog-with-okta-ldap.png)
+
+   </div>
+
+   A successful sign-in returns the following dialog:
+
+   <div class="full border">
+
+   ![Successful sign-in dialog saying Welcome home user01](/img/architecture/directory-coexistence/ldap-to-okta-signin-to-ldap-directory.png)
+
+   </div>
+
+### Troubleshooting
+
+If you see an error message similar to the following when you sign in:
+
+<div class="half border">
+
+![Sign-in nested exception root exception error](/img/architecture/directory-coexistence/ldap-to-okta-error-didnt-update-okta-domain.png)
+
+</div>
+
+perform the following steps:
+
+1. Press <kbd>Ctrl</kbd> + <kbd>C</kbd> in the console to stop the containers.
```

Comment:
```suggestion
1. Enter <kbd>Ctrl</kbd> + <kbd>C</kbd> in the terminal to stop the containers.
```

Reply:
Goign to leave as console for now. Lots of instances of console elsewhere.